test/facet.js

```diff
@@ -8,6 +8,9 @@ var Facet = require('../').models.Facet;
 
 describe('Facet', function() {
   describe('Facet.create(def, cb)', function() {
+    this.timeout(15000);
+
+    before(resetWorkspace);
     beforeEach(givenBasicWorkspace);
```

Comment:
Isn't `givenBasicWorkspace` calling `resetWorkspace` under the hood? I find this change little bit suspicious, but if it fixes CI failures then I don't object.